odoorpc/rpc/jsonrpclib.py

```diff
@@ -120,6 +126,30 @@ def __call__(self, url, params=None):
         )
         return result
 
+    def _get_response(self, request):
+        if not self.auto_retry:
+            return self._opener.open(request, timeout=self._timeout)
+
+        stop = False
+        sleep_base = self.auto_retry_step
+        iteration = 1
+
+        while not stop:
+            try:
+                response = self._opener.open(request, timeout=self._timeout)
+                return response
+            except HTTPError as e:
+                if e.code == 429:
+                    sleep_time = math.pow(sleep_base, iteration)
+                    logger.debug('Error "Too Many Requests", retrying in ' + str(sleep_time))
+                    sleep(sleep_time)
+                    if iteration == self.auto_retry_max:
+                        raise(e)
+                    else:
+                        iteration = iteration + 1
+                else:
+                    raise(e)
```

Comment:
Some syntactical enhancements:

```suggestion
                        raise
                    else:
                        iteration += 1
                else:
                    raise
```